src/components/createRecipe/recipeForm/Step1.jsx

```diff
@@ -14,22 +14,20 @@ import {
   Addtitle,
   Section2,
   Title,
-  SwitchDiv,
-  ButtonPointer
 } from "./FormStyled.styles";
 
-const useStyles = makeStyles(theme => ({
-  root: {
-    display: 'flex',
-    flexWrap: 'wrap',
-  },
-  textField: {
-    marginLeft: theme.spacing(1),
-    marginRight: theme.spacing(1),
-    color: 'white'
-    // font-size: 18px
-  },
-}));
+// const useStyles = makeStyles(theme => ({
+//   root: {
+//     display: 'flex',
+//     flexWrap: 'wrap',
+//   },
+//   textField: {
+//     marginLeft: theme.spacing(1),
+//     marginRight: theme.spacing(1),
+//     color: 'white'
+//     // font-size: 18px
+//   },
+// }));
```

Comment:
What's up with this one? Is it a Material UI thing?

Reply:
just normal styling but never used